andy/src/test/java/selenium/SecurityExploitTests.java

```diff
@@ -9,6 +10,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static unit.writer.standard.StandardResultTestAssertions.*;
 
+@Tag("ci")
 public class SecurityExploitTests extends WebLabSeleniumTestBase {
```

Comment:
Those tests aren't meant to run only in CI. Do they fail on your machine?

Reply:
Still not working here. I’ll merge this anyways. This isn’t a test that I feel like we have to run locally, let the CI do it. Also, note that the tag only gets disabled if you manually do it, I didn’t change the maven script.